internal/impl/mongodb/cdc/input.go

```diff
@@ -669,6 +680,27 @@ func (m *mongoCDC) readSnapshotRange(
 	return nil
 }
 
+func (m *mongoCDC) getCurrentResumeToken(ctx context.Context) (bson.Raw, error) {
+	filter := []bson.M{{"$match": bson.M{
+		"ns.coll": bson.M{"$in": slices.Clone(m.collections)},
```

Comment:
Why do we need clone here?

Reply:
it cannot be modified l - it can be removed